### PR TITLE
Vulkan : add logic op

### PR DESCRIPTION
**Pull request by raven02**
All backend should have Logic Op implemented.

### CODE REVIEW

File: rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp [hunk 1]
```diff
@@ -128,7 +128,6 @@ D3D12_COMPARISON_FUNC get_compare_func(u32 op)
 	case CELL_GCM_NOTEQUAL: return D3D12_COMPARISON_FUNC_NOT_EQUAL;
 	case CELL_GCM_GEQUAL: return D3D12_COMPARISON_FUNC_GREATER_EQUAL;
 	case CELL_GCM_ALWAYS: return D3D12_COMPARISON_FUNC_ALWAYS;
-	case CELL_GCM_ZERO:
 		break;
```
Comment (kd-11) [inline rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp]: This break is useless if the case has been removed. It is in fact unreachable

Comment (raven02) [inline rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp]: Yes , i miss out . Will remove it.

**Comment by raven02** [comment]
Done and rebased .

**Comment by kd-11** [comment]
@raven02 maybe squash those 2 extra commits? It'll help clean up the commit log

**Comment by raven02** [comment]
Humm i got something wrong when squash the last 2 commits .Probably let me pull a new one.

**Pull request closed without merging**